Module/utm/vis_jedikey.utm.json

```diff
@@ -0,0 +1,139 @@
+{
+  "__data_type": "UTM ",
+  "BlackMarket": {
+    "type": "byte",
+    "value": 0
+  },
+  "BM_MarkDown": {
+    "type": "int",
+    "value": 25
+  },
+  "Comment": {
+    "type": "cexostring",
+    "value": ""
+  },
+  "ID": {
+    "type": "byte",
+    "value": 5
+  },
+  "IdentifyPrice": {
+    "type": "int",
+    "value": -1
+  },
+  "LocName": {
+    "type": "cexolocstring",
+    "value": {
+      "0": "Viscara Temple Keys"
+    }
+  },
+  "MarkDown": {
+    "type": "int",
+    "value": 0
```

Comment:
MarkDown should be 40. MarkUp should be 100 for all stores. If you don't do a mark down players will sell to these stores for no loss.

Reply:
Make sure you update instances after modifying this blueprint. Just changing it here doesn't change the ones you placed in-world.